.github/workflows/delta.yaml

```
@@ -133,15 +126,46 @@ jobs:
+            )
+            podman save "$repo:$name" | zstd -o $name.tar.zstd
+            echo "name=$name" >> $GITHUB_OUTPUT
+            echo "path=$name.tar.zstd" >> $GITHUB_OUTPUT
+
+      - name: Upload image artifact
+        if: ${{ !inputs.push }}
+        uses: actions/upload-artifact@v4
         with:
-          workflow: manifest.yaml
-          token: ${{ secrets.DISPATCH_PAT }}
+          name: ${{ steps.export.outputs.name }}
```

actionlint (1.7.8)

property "name" is not defined in object type {updates: string}

(expression)